ibis/backends/__init__.py

```diff
@@ -477,7 +478,7 @@ def to_parquet(
     def to_parquet_dir(
         self,
         expr: ir.Table,
-        directory: str | Path,
+        path: str | Path,
```

Comment:
The use of `directory` was an ask in the `to_parquet_dir` PR, see https://github.com/ibis-project/ibis/pull/9781#discussion_r1709931796 . DO we want to go backwards on that?